InvenTree/order/api.py

```diff
@@ -114,7 +116,7 @@ class PurchaseOrderList(APIDownloadMixin, generics.ListCreateAPIView):
 
     def create(self, request, *args, **kwargs):
         """Save user information on create."""
-        serializer = self.get_serializer(data=request.data)
+        serializer = self.get_serializer(data=self.clean_data(request.data))
```

Comment:
Nice work catching this, I was concerned that any custom calls to `create` may have been missed.

Reply:
@SchrodingersGat I went through every single method called create or update and each line in those